docs/source/errors.mdx

```diff
@@ -4,13 +4,18 @@ description: Learn the error codes you can receive if composition fails.
 toc: false
 ---
 
-When Apollo Gateway attempts to compose the schemas provided by your [subgraphs](./building-supergraphs/subgraphs-overview/) into a supergraph schema, it confirms that:
+When Apollo Federation attempts to compose the schemas provided by your [subgraphs](./building-supergraphs/subgraphs-overview/) into a supergraph schema, it confirms that:
```

Comment:
```suggestion
When GraphOS attempts to compose the schemas provided by your [subgraphs](./building-supergraphs/subgraphs-overview/) into a supergraph schema, it confirms that:
```

Reply:
As I understand it, Apollo Federation is the architecture, not the thing doing composition. Would GraphOS work here?

Reply:
Yea, this works better to recognize the entire ecosystem